zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -39,6 +43,8 @@ object Server {
     maxRequestSize: Int = 4 * 1024, // 4 kilo bytes
     error: Option[Throwable => ZIO[R, Nothing, Unit]] = None,
     sslOption: ServerSSLOptions = null,
+    hostname: String = null,
+    inetAddress: InetAddress = null,
```

Comment:
Any reason not to use `Option` instead of `null`?

```suggestion
    hostname: Option[String] = None,
    inetAddress: Option[InetAddress] = None,
```